Breaking change on LanguageGeneration package in 4.8.0-prevew builds

#### What project is affected?
Virtual Assistant Sample & tests

#### What language is this in?
C#

#### What happens?
Unit Tests project fails with the following type errors in the `Microsoft.Bot.Builder.LanguageGeneration` package:

```
InterruptionTests.cs(18,65): error CS7069: Reference to type 'TemplateEngine' claims it is defined in 'Microsoft.Bot.Builder.LanguageGeneration', but it could not be found [D:\a\1\s\samples\csharp\assistants\virtual-assistant\VirtualAssistantSample.Tests\VirtualAssistantSample.Tests.csproj]
InterruptionTests.cs(31,64): error CS7069: Reference to type 'TemplateEngine' claims it is defined in 'Microsoft.Bot.Builder.LanguageGeneration', but it could not be found [D:\a\1\s\samples\csharp\assistants\virtual-assistant\VirtualAssistantSample.Tests\VirtualAssistantSample.Tests.csproj]
InterruptionTests.cs(45,65): error CS7069: Reference to type 'TemplateEngine' claims it is defined in 'Microsoft.Bot.Builder.LanguageGeneration', but it could not be found [D:\a\1\s\samples\csharp\assistants\virtual-assistant\VirtualAssistantSample.Tests\VirtualAssistantSample.Tests.csproj]
InterruptionTests.cs(46,62): error CS7069: Reference to type 'TemplateEngine' claims it is defined in 'Microsoft.Bot.Builder.LanguageGeneration', but it could not be found [D:\a\1\s\samples\csharp\assistants\virtual-assistant\VirtualAssistantSample.Tests\VirtualAssistantSample.Tests.csproj]
InterruptionTests.cs(59,64): error CS7069: Reference to type 'TemplateEngine' claims it is defined in 'Microsoft.Bot.Builder.LanguageGeneration', but it could not be found [D:\a\1\s\samples\csharp\assistants\virtual-assistant\VirtualAssistantSample.Tests\VirtualAssistantSample.Tests.csproj]
LocalizationTests.cs(24,68): error CS7069: Reference to type 'TemplateEngine' claims it is defined in 'Microsoft.Bot.Builder.LanguageGeneration', but it could not be found [D:\a\1\s\samples\csharp\assistants\virtual-assistant\VirtualAssistantSample.Tests\VirtualAssistantSample.Tests.csproj]
LocalizationTests.cs(51,68): error CS7069: Reference to type 'TemplateEngine' claims it is defined in 'Microsoft.Bot.Builder.LanguageGeneration', but it could not be found [D:\a\1\s\samples\csharp\assistants\virtual-assistant\VirtualAssistantSample.Tests\VirtualAssistantSample.Tests.csproj]
LocalizationTests.cs(78,68): error CS7069: Reference to type 'TemplateEngine' claims it is defined in 'Microsoft.Bot.Builder.LanguageGeneration', but it could not be found [D:\a\1\s\samples\csharp\assistants\virtual-assistant\VirtualAssistantSample.Tests\VirtualAssistantSample.Tests.csproj]
LocalizationTests.cs(105,68): error CS7069: Reference to type 'TemplateEngine' claims it is defined in 'Microsoft.Bot.Builder.LanguageGeneration', but it could not be found [D:\a\1\s\samples\csharp\assistants\virtual-assistant\VirtualAssistantSample.Tests\VirtualAssistantSample.Tests.csproj]
LocalizationTests.cs(132,68): error CS7069: Reference to type 'TemplateEngine' claims it is defined in 'Microsoft.Bot.Builder.LanguageGeneration', but it could not be found [D:\a\1\s\samples\csharp\assistants\virtual-assistant\VirtualAssistantSample.Tests\VirtualAssistantSample.Tests.csproj]
MainDialogTests.cs(33,65): error CS7069: Reference to type 'TemplateEngine' claims it is defined in 'Microsoft.Bot.Builder.LanguageGeneration', but it could not be found [D:\a\1\s\samples\csharp\assistants\virtual-assistant\VirtualAssistantSample.Tests\VirtualAssistantSample.Tests.csproj]
MainDialogTests.cs(46,65): error CS7069: Reference to type 'TemplateEngine' claims it is defined in 'Microsoft.Bot.Builder.LanguageGeneration', but it could not be found [D:\a\1\s\samples\csharp\assistants\virtual-assistant\VirtualAssistantSample.Tests\VirtualAssistantSample.Tests.csproj]
MainDialogTests.cs(59,65): error CS7069: Reference to type 'TemplateEngine' claims it is defined in 'Microsoft.Bot.Builder.LanguageGeneration', but it could not be found [D:\a\1\s\samples\csharp\assistants\virtual-assistant\VirtualAssistantSample.Tests\VirtualAssistantSample.Tests.csproj]
MainDialogTests.cs(60,62): error CS7069: Reference to type 'TemplateEngine' claims it is defined in 'Microsoft.Bot.Builder.LanguageGeneration', but it could not be found [D:\a\1\s\samples\csharp\assistants\virtual-assistant\VirtualAssistantSample.Tests\VirtualAssistantSample.Tests.csproj]
OnboardingDialogTests.cs(26,64): error CS7069: Reference to type 'TemplateEngine' claims it is defined in 'Microsoft.Bot.Builder.LanguageGeneration', but it could not be found [D:\a\1\s\samples\csharp\assistants\virtual-assistant\VirtualAssistantSample.Tests\VirtualAssistantSample.Tests.csproj]
OnboardingDialogTests.cs(27,65): error CS7069: Reference to type 'TemplateEngine' claims it is defined in 'Microsoft.Bot.Builder.LanguageGeneration', but it could not be found [D:\a\1\s\samples\csharp\assistants\virtual-assistant\VirtualAssistantSample.Tests\VirtualAssistantSample.Tests.csproj]

```

#### What are the steps to reproduce this issue?
1. Update all packages in Virtual Assistant Sample to `v4.8.0-preview*`
1. Run `dotnet build`

#### What were you expecting to happen?
`dotnet build` works successfully

#### Can you share any logs, error output, etc.?
[Relevant build logs](https://fuselabs.visualstudio.com/Solutions/_build/results?buildId=105509&view=logs&j=fd490c07-0b22-5182-fac9-6d67fe1e939b&t=5ce004ff-f36e-5243-994b-d03ca51fb277)
#### Any screenshots or additional context?

--- a/build/build.cognitive-models.ps1
+++ b/build/build.cognitive-models.ps1
@@ -33,8 +33,8 @@ $languageModel = [pscustomobject]@{
 $knowledgeBase = [pscustomobject]@{
     endpointKey = ""
     id = ""
-    host = ""
-    knowledgeBaseId = ""
+    hostname = ""
+    kbId = ""
     name = ""
     subscriptionKey = ""
 }

--- a/samples/csharp/assistants/virtual-assistant/VirtualAssistantSample/Services/BotServices.cs
+++ b/samples/csharp/assistants/virtual-assistant/VirtualAssistantSample/Services/BotServices.cs
@@ -54,12 +54,12 @@ namespace VirtualAssistantSample.Services
                 {
                     var qnaEndpoint = new QnAMakerEndpoint()
                     {
-                        KnowledgeBaseId = kb.KnowledgeBaseId,
+                        KnowledgeBaseId = kb.KbId,
                         EndpointKey = kb.EndpointKey,
-                        Host = kb.Host,
+                        Host = kb.Hostname,
                     };
 
-                    set.QnAConfiguration.Add(kb.KnowledgeBaseId, qnaEndpoint);
+                    set.QnAConfiguration.Add(kb.Id, qnaEndpoint);
                 }
 
                 CognitiveModelSets.Add(language, set);

--- a/samples/csharp/assistants/virtual-assistant/VirtualAssistantSample/VirtualAssistantSample.csproj
+++ b/samples/csharp/assistants/virtual-assistant/VirtualAssistantSample/VirtualAssistantSample.csproj
@@ -29,7 +29,7 @@
     <PackageReference Include="Microsoft.Bot.Configuration" Version="4.8.0-preview-200213-105928" />
     <PackageReference Include="Microsoft.Bot.Connector" Version="4.8.0-preview-200213-105928" />
     <PackageReference Include="Microsoft.Bot.Schema" Version="4.8.0-preview-200213-105928" />
-    <PackageReference Include="Microsoft.Bot.Solutions" Version="0.9.0-daily228" />
+    <PackageReference Include="Microsoft.Bot.Solutions" Version="0.9.0-daily239" />
   </ItemGroup>
 
   <ItemGroup>

--- a/samples/csharp/assistants/virtual-assistant/VirtualAssistantSample/cognitivemodels.json
+++ b/samples/csharp/assistants/virtual-assistant/VirtualAssistantSample/cognitivemodels.json
@@ -25,9 +25,9 @@
       "knowledgebases": [
         {
           "endpointKey": "",
-          "host": "",
+          "hostname": "",
           "id": "",
-          "knowledgeBaseId": "",
+          "kbId": "",
           "name": "",
           "subscriptionKey": ""
         }

